Documentation/Plus Codes Specification.md

```diff
@@ -0,0 +1,119 @@
+# Plus Codes Specification
+
+## Version history
+
+All substantial changes changes to this document are listed here.
+
+* Version 1.1.0 / 2021-06-04 / [William Entriken](https://github.com/fulldecent/)
+  * Separate Plus Codes specification from [Open Location Code API specification](./Open Location Code API Specification.md)
+  * Define codes for north and south poles
+  * Specify which code levels of full codes may be shortened to which code levels of short codes
+  * Establish consistent wording/naming
+* Version 1.0.0 / 2019-04-29 / [Doug Rinckes](https://github.com/drinckes), Google / Philipp Bunge, Google
+  * Initial public release
+
+## Area and bounds
+
+A **Plus Code** represents a locus of coordinates ("**area**") with **bounds** on northern & southern parallels and western & eastern meridians. This area includes exactly the coordinates:
+
+1. Inside (excluding) the bounds;
+2. On the western bound between (excluding) the northern and southern bounds;
+3. (If the northern bound is 90° N and the western bound is 0° W) the North Pole (i.e. the point at 90° N);
+4. (If the southern bound is 90° S and the western bound is 0° W) the South Pole (i.e. the point at 90° S); and
+5. (If the southern bound is not 90° S) on the southern bound from (including) the western bound to (excluding) the eastern bound.
+
+This specification references latitudes and longitudes on Earth under [WGS 84](https://earth-info.nga.mil), which is the standard used by the Global Positioning System. If another geodetic is used, that context MUST be agreed between the producer and consumer of a Plus Code.
+
+The **center** of a Plus Code area is defined as the arithmetic mean of its bound's parallels and meridians.
+
+## Precision
```

Comment:
Comment - this is not a problem with this PR, but still:

1. Is "Precision" even the correct term in this context, or should it be "Accuracy"?
2. Is it actually correct to give precision/accuracy as the "maximum side length", or should this rather be something like the diagonal (or even half of that, assuming that we're talking about distance to center?).

Reply:
Agreed, fixed at 33044f51f05234c96da1600ed24db9b5f46100a7